telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java

```diff
@@ -11,14 +11,31 @@
  * <a href="https://core.telegram.org/bots/api#getupdates">long-polling</a> method
  */
 public abstract class TelegramLongPollingBot extends DefaultAbsSender implements LongPollingBot {
+    /**
+     * If this is used getBotToken has to be overridden in order to return the bot token!
+     * @deprecated Overwriting the getBotToken() method is deprecated. Use the constructor instead
+     */
+    @Deprecated()
     public TelegramLongPollingBot() {
         this(new DefaultBotOptions());
     }
 
+    /**
+     * If this is used getBotToken has to be overridden in order to return the bot token!
+     * @deprecated Overwriting the getBotToken() method is deprecated. Use the constructor instead
+     */
+    @Deprecated()
     public TelegramLongPollingBot(DefaultBotOptions options) {
         super(options);
     }
 
+    public TelegramLongPollingBot(String botToken) {
+        this(new DefaultBotOptions(), botToken);
+    }
+    public TelegramLongPollingBot(DefaultBotOptions options, String botToken) {
+        super(options, botToken);
+    }
+
```

Comment:
need to add translation after line 34

Reply:
What do you mean?

Reply:
[HERE](https://ibb.co/FKD3pB2)

Reply:
What you you mean by translations

Reply:
\n character

Reply:
Oh a line break. Yeah there's an empty line missing between the methods

Reply:
Thanks. I will remember :)